setup.py

```diff
@@ -121,14 +121,14 @@
     # optional dependencies
     "apache-beam>=2.26.0",
     "elasticsearch",
-    "aiobotocore==1.2.2",
-    "boto3==1.16.43",
-    "botocore==1.19.52",
+    "aiobotocore",
+    "boto3",
+    "botocore",
     "faiss-cpu",
     "fsspec[s3]",
     "moto[s3,server]==2.0.4",
     "rarfile>=4.0",
-    "s3fs",
+    "s3fs==2021.08.1",
```

Comment:
Is it necessary to pin this version? I guess this will have to be eventually updated/removed to allow future versions of `s3fs`...

Reply:
This is useful to get the right versions of aiobotocore/boto3/botocore. If we don't set a minimum version, then pip would pick a version that is too old (s3fs 0.5.3). If we set a minimum version without pinning the exact version, the pip resolver takes forever to find the right versions for s3fs/aiobotocore/boto3/botocore (>10min).

Maybe an alternative would be to have minimum versions for all of them, but I haven't tested